CBLClient/Apps/CBLTestServer-Dotnet/TestServer/ValueSerializer.cs

```diff
@@ -38,9 +38,15 @@ public static string Serialize(Object value, Type t)
             {
                 return "L" + value;
             }
-            else if (t.Equals(typeof(float)) || t.Equals(typeof(double)))
+            else if (t.Equals(typeof(float)))
```

Comment:
this is required only in .Net to have differrent if condition of Float and double. Other platforms we combine. Hope Jim confirmed this.

Reply:
Yes other platforms have different conditions